scripts/unpublish.js

```diff
@@ -0,0 +1,87 @@
+const { readFile, readdir } = require('node:fs').promises;
+const { join } = require('node:path');
+const util = require('node:util');
+const { compare } = require('compare-versions');
```

Comment:
You need to include this pacakge also here?

Reply:
nice catch
https://github.com/FuelLabs/fuels-npm-packs/pull/61/commits/b2d1725894e0a9f7ff3f12fd0d7f1e42a9f0ebb6